civihr_employee_portal/features/node_export_files/mydetails_contact_details.export

```diff
@@ -0,0 +1,368 @@
+array(
```

Comment:
@powdevel what is this used for ? why we are adding export file manually ?

Reply:
@hitesh-jain this file is used to create a webform. I'm following current patterns in `civihr_employee_portal`. Files inside that folder are created or refreshed on updates using `drush_civihr_employee_portal_refresh_node_export_files`. See `civihr_employee_portal.install` for examples.

Reply:
@mickadoo Using features is the best way to export files and config. I'm certainly not sure why and how they are handling export in civihr as it seems to be that they are exporting manually using .export file I think.

Reply:
@hitesh-jain for now it can be exported this way, which is not the best way. Unfortunately this task has taken a lot of time. In our last talk with Mike, Alessandro and Debarshi we agreed to let this issue to be solved in another task, just for time sake.